entity-management-web/src/test/java/eu/europeana/entitymanagement/validation/EntityFieldsValidatorTest.java

```diff
@@ -43,33 +45,54 @@
 
 
     @Test
-    public void validateEntityFieldsForOrganization() throws JsonMappingException, JsonProcessingException, IOException {
+    public void validateEntityFieldsOrganization() throws JsonMappingException, JsonProcessingException, IOException {
 	//TODO: the input file must be updated to comply with the correct jsonld serialization
         Organization organization = objectMapper.readValue(loadFile(ORGANIZATION_VALIDATE_FIELDS_JSON), Organization.class);
-        EntityRecord entityRecord1 = new EntityRecord();
-        entityRecord1.setEntity(organization);
-        entityRecord1.setEntityId(organization.getEntityId());        
-        Set<ConstraintViolation<Entity>> violations = emValidatorFactory.getValidator().validate(entityRecord1.getEntity());
+        EntityRecord entityRecord = new EntityRecord();
+        entityRecord.setEntity(organization);
+        entityRecord.setEntityId(organization.getEntityId());        
+        Set<ConstraintViolation<Entity>> violations = emValidatorFactory.getValidator().validate(entityRecord.getEntity(), EntityFieldsCompleteValidatorGroup.class);
         for (ConstraintViolation<Entity> violation : violations) {
             System.out.println(violation.getMessageTemplate());
         }   
         //TODO: remove constraine violation: "The entity fields values are valid."
         Assertions.assertEquals(18, violations.size());
+        /*
+         * checking the minimal level of validation
+         */
+        violations.clear();
+        violations = emValidatorFactory.getValidator().validate(entityRecord.getEntity(), EntityFieldsMinimalValidatorGroup.class);
```

Comment:
the mimimal validation must be implemented in own integration test

Reply:
Done.